src/universal/redux/AuthEngine.js

```diff
@@ -6,7 +6,7 @@ export default class AuthEngine {
     this.reducerName = reducerName;
   }
   saveToken(name, token, options, callback) {
```

Comment:
`options`?

Reply:
that's what socket cluster calls it.

Reply:
Sorry that was a crappy comment on my part, haha! I think I was wondering why it's a parameter if it's not being used. Is this class fulfilling an interface for a 3rd party library or something?